src/components/commons/cards/CommonCard.tsx

```diff
@@ -0,0 +1,66 @@
+import Image from 'next/image';
+import Link from 'next/link';
+
+import classNames from 'classnames/bind';
+
+import { SVGS } from '@/constants';
+import { getFormatDate, toFixedOneDecimal } from '@/utils';
+
+import StarRating from '@/components/commons/StarRating';
+import Tag from '@/components/commons/Tag';
+
+import { PostTypes } from '@/types';
+
+import styles from './CardCommonStyle.module.scss';
+
+const cx = classNames.bind(styles);
+const { location, calendar } = SVGS;
+
+export type CommonCardProps = {
+  path: string;
+  postType: PostTypes;
+  title: string;
+  address: string;
+  rating: number;
+  reviewCount: number;
+  createdAt: string;
+};
+
+const CommonCard = ({ path, postType, title, address, rating, reviewCount, createdAt }: CommonCardProps) => {
+  const isOffline = postType === 'offline';
+
+  return (
+    <article className={cx('card')}>
+      <Link href={`${path}`} className={cx('card-inner')}>
```

Comment:
```suggestion
      <Link href={path} className={cx('card-inner')}>
```
path가 string 이니까 바로 넣어줘도 좋을 거 같습니다

Reply:
네! 반영했습니다~